src/components/performances/PerformancesSection.jsx

```diff
@@ -37,29 +39,33 @@ class PerformancesSection extends React.Component {
   }
 
   render() {
-    const headings = ['Dance Title', 'Dance Entry', 'School', 'Acaademic Level', 'Level of Competition', 'Dance Style', 'Dance Size'];
     const { performances } = this.state;
     const { match: { params: { eventId }}} = this.props;
-    const collectionName = `events/${eventId}/performances`;
+    const headings = ['Dance Title', 'Dance Entry', 'School', 'Acaademic Level', 'Level of Competition', 'Dance Style', 'Dance Size'];
+    const showPerformances = Array.isArray(performances) && performances.length > 0;
 
     return (
       <React.Fragment>
         <SectionHeader eventId={eventId} title="performance" />
+        <TableSettings />
         <Table>
           <TableHeader headings={headings} />
           <TableBody>
-            {(Array.isArray(performances) && performances.length)
-              && performances.map(performance => (
+            {showPerformances && performances.map((performance) => {
```

Comment:
I'm concerned about this logic actually, if we do showPerformances = true but performances.length = 0, then nothing shows up here.